lib/fishjam/cluster/room_service.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Fishjam.Cluster.RoomService do
+  @moduledoc """
+  Module responsible for managing rooms in the entire cluster.
+  """
+
+  @behaviour Fishjam.RoomService
+
+  require Logger
+
+  alias Fishjam.{FeatureFlags, Room, RPCClient}
+
+  @local_module Fishjam.Local.RoomService
+
+  @impl true
+  def create_room(config) do
+    node_resources = RPCClient.multicall(@local_module, :get_resource_usage)
+    min_node = find_best_node(node_resources)
+
+    if node_resources == [],
```

Comment:
I wouldnt write if for case like that, this shouldn't ever happen -> call to self should always succeed so there will be at least 1 element on list right?

Reply:
I agree but @Rados13 wanted to handle this case